README.md

```diff
@@ -1 +1,54 @@
-# GatorGrade
+# GatorGrade: A Python Tool to Implement GatorGrader
+
+GatorGrade is a Python tool that executes GatorGrader, an automatic grading tool that can be used to check assignments through user-created checks. GatorGrade is the newer Python-based version of GatorGradle which can be found on GitHub at this link: https://github.com/GatorEducator/gatorgradle/blob/master/README.md. GatorGrade takes in a gatorgrader.yml file and tests each check for correctness against the code that it is connected to.
+
+## Table of Contents:
+
+- Installing GatorGrade
+  - Including GatorGrade in Your Project
+- Using GatorGrade
+  - Running Checks
+    - Interpreting Output (Output Team)
+  - Generating a gatorgrade.yml File (Generate Team)
+    - Configuring GatorGrade Checks (Input Team)
+- Contributing to GatorGrade
+  - Installing Dev Environment
+- Contributors
```

Comment:
There is a VSCode extension that automatically generates a table of contents and adds the links for all the sections, which are missing right now. Using it might make it much easier to work with the table and making sure it's always updated and nicely formatted. [Here is the link](https://marketplace.visualstudio.com/items?itemName=fantasy.markdown-all-in-one-for-web)

Reply:
Thank you for the suggestion I will see to that now